package.json

```diff
@@ -14,8 +15,11 @@
     "@cypress/grep": "^4.0.1",
     "cypress": "^13.7.1",
     "cypress-mochawesome-reporter": "^3.8.2",
+    "cypress-xpath": "^2.0.1",
     "dotenv": "^16.4.5",
     "eslint": "^8.57.0",
+    "eslint-gitignore": "^0.1.0",
+    "eslint-plugin-import": "^2.29.1",
     "mocha": "^10.3.0"
   }
-}
+}
```

Comment:
GitHub does some checks and one of them is the missed blank line at the end of the file. It can be added to the lint/formatting tools.